lib/identity_config.rb

```diff
@@ -231,6 +231,7 @@ def self.build_store(config_map)
     config.add(:in_person_email_reminder_final_benchmark_in_days, type: :integer)
     config.add(:in_person_email_reminder_late_benchmark_in_days, type: :integer)
     config.add(:in_person_proofing_enabled, type: :boolean)
+    config.add(:in_person_enable_public_address_search, type: :boolean)
```

Comment:
```suggestion
    config.add(:in_person_public_address_search_enabled, type: :boolean)
```
to be consistent with the naming of most boolean feature flags